puppet/modules/web/manifests/vhost/debugs.pp

```diff
@@ -1,20 +0,0 @@
-# @summary Set up the debugs vhost
-# @api private
-class web::vhost::debugs(
-  Hash[String, Hash] $htpasswds = {},
-) {
-  web::vhost { 'debugs':
-    docroot_owner => 'nobody',
-    docroot_group => 'nobody',
-    docroot_mode  => '0755',
-    attrs         => {
-      'custom_fragment' => template('web/debugs.conf.erb'),
-    },
-  }
-
-  $htpasswds.each |$username, $options| {
```

Comment:
Since this is removed, `web::htpasswd` is unused and `puppet/modules/web/manifests/htpasswd.pp` can be dropped. It also means we can drop the related bits (`citrin/htpasswd` in `Puppetfile` and `puppet/spec/defines/web_htpasswd_spec.rb`).